### DISCUSSION TITLE
Multiple selections for `RelationalFieldConditionRule`

### DISCUSSION THREAD

**Discussion by gglnx** [craftcms/cms#12538] [Ideas] [answered]
Would it be possible to allow multiple elements to be selected (with `OR`) for relational field conditions? `TypeConditionRule` supports this.

**Comment by brandonkelly** [comment, marked as answer]
Not currently possible since they go through the relational field’s query param (same as doing `craft.entries().myRelField(...)`), which can only have one value. But we’re thinking about ways to improve this.

**Reply by gglnx** [reply]
I wrote a replacement (injected via `ElementCondition::EVENT_REGISTER_CONDITION_RULE_TYPES`) which replaces `RelationalFieldConditionRule` with a version which supports multiple selections as `craft.entries().myRelField(...)` supports `['or', 1, 2, …]` queries https://gist.github.com/gglnx/b771b22185a59ba9e90174f5b3f07d6b
